packages/build-utils/src/convert-runtime-to-plugin.ts

```diff
@@ -141,12 +214,15 @@ export function convertRuntimeToPlugin(
         'pages',
         `${entrypoint}.nft.json`
       );
+
       const json = JSON.stringify({
         version: 1,
-        files: tracedFiles.map(f => ({
-          input: normalizePath(relative(nft, f.absolutePath)),
-          output: normalizePath(f.relativePath),
-        })),
+        files: tracedFiles.map(file => {
+          return {
+            input: normalizePath(join('../../..', traceDirInputs, file)),
```

Comment:
I don't think this will work.

The reason why it was relative before, is because the file can be nested, e.g. `/api/users/info.py`, in that case it'd need to be `../../../../`, that's why `relative` was used.

Reply:
Oh, right, I missed that.

Unfortunately `relative` adds one level too much, but I'll pick a different solution then!

Reply:
Seems like my other changes made `relative` work properly again! 🤔 

Fixed in https://github.com/vercel/vercel/pull/7117/commits/ca890767bc574df2a58bb48fd0dc6ecc5a096d5b and https://github.com/vercel/vercel/pull/7117/commits/7c4baeaafaf41609f47c97a09f5e9647fd8b89ee.